src/main/java/se/liu/ida/hefquin/engine/wrappers/graphqlwrapper/GraphQL2RDFConfiguration.java

```diff
@@ -0,0 +1,62 @@
+package se.liu.ida.hefquin.engine.wrappers.graphqlwrapper;
+
+import java.util.Set;
+
+import se.liu.ida.hefquin.engine.federation.GraphQLEndpoint;
+import se.liu.ida.hefquin.engine.wrappers.graphqlwrapper.data.impl.GraphQLFieldType;
+
+/**
+ * Used to represent and retrieve information about a virtual RDF view for a GraphQL endpoint
+ */
+public interface GraphQL2RDFConfiguration {
+
+    /**
+     * Verifies that @param uri that correponds to a class exists for @param endpoint
```

Comment:
My confusion continues. Now there seem to be URIs that may exist for GraphQL endpoints. How? / "exists" in what sense? 

...and how is "verifies" meant? In the sense that objects of classes that implement this interface can perform such a check? If so, then it sounds more like a helper function to me and I wonder why such a helper function is part of something that is called a "configuration"?